.travis.yml

```diff
@@ -43,16 +43,15 @@ jobs:
       - yarn lint
       - yarn depcheck
       - yarn test
-      - yarn chromatic --exit-zero-on-changes --app-code="dlpro96xybh" # v1 for cross-browser
-      - yarn chromatic --exit-zero-on-changes # v2 for visual review features and Storybook hosting
+      - yarn chromatic --quiet --exit-zero-on-changes
```

Comment:
Added the `--quiet` option to decrease the noise in the log output from `yarn chromatic` where it runs `yarn build-storybook` which invokes `webpack`. Normally Webpack overwrites the last line in a log and we don't see a massive amount of output. In CI, a log line cannot be rewritten, so it ends up noisy.